### PR TITLE
Add hc styling to fix Toggle control

### LINKED ISSUE #12298: Fabric toggle control does not support high contrast
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out if relevant)

### Describe the issue:

Currently the toggle does not support high contrast.

#### Actual behavior:

The background of the toggle should be cyan when on (or whatever the user setting is for high contrast). 

#### Expected behavior:

Instead, the background of the toggle is black. 

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)
High

Products/sites affected: (if applicable)

### PR DESCRIPTION
**Pull request by tomi-msft**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #12298 
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Toggle component does not support HC styling in Edge Chromium. This fix adds styling that will allow Toggle to adopt HC styling.

**Before**
image

**After**
image

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by dzearing** [comment]
It seems like the focus rectangle is off. It may not have been caused by this PR, but there is a tiny dot in the thumb of the toggle which I think is the focus rect.

File: packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts [hunk 1]
```diff
@@ -129,8 +134,9 @@ export const getStyles = (props: IToggleStyleProps): IToggleStyles => {
                 },
               ],
               [HighContrastSelector]: {
-                backgroundColor: 'WindowText',
+                backgroundColor: 'Highlight',
```
Comment (xugao) [inline packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts:137]: could you make the similar fix to `toggle.scss` in `react-next`?
we should add the styles in `.pill &.checked` section and `@include high-contrast-edge-chromium-no-adjust;` is how you achieve the same as `getEdgeChromiumNoHighContrastAdjustSelector`

File: change/@fluentui-react-next-2020-08-04-09-58-27-a11y-toggle.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "update Toggle in react-next package",
```
Comment (xugao) [inline change/@fluentui-react-next-2020-08-04-09-58-27-a11y-toggle.json:3]:
```suggestion
  "comment": "Toggle: fix pill HC style when selected",
```

File: change/office-ui-fabric-react-2020-08-03-14-14-33-a11y-toggle.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Add hc styling to fix toggle",
```
Comment (xugao) [inline change/office-ui-fabric-react-2020-08-03-14-14-33-a11y-toggle.json:3]:
```suggestion
  "comment": "Toggle: fix pill HC style when selected",
```

**Comment by xugao** [comment]
to David's comment, i think the issue is with this line [here](https://github.com/microsoft/fluentui/blob/master/packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts#L179). I think it should be `borderWidth: DEFAULT_THUMB_SIZE / 2` instead.

@tomi-msft - could you confirm my theory and issue a separate PR to fix? thanks!

**Pull request merged**